_i18n/hu/_posts/2020-05-05-CYBERG_IT_Irasbeli_szavazas.md

```diff
@@ -0,0 +1,7 @@
+---
+# layout: news
+title: CYBERG_IT_Irasbeli_szavazas
+docs:
+  - file: CYBERG_IT_Irasbeli_szavazas_20200424_jkv.pdf.pdf
```

Comment:
please do not use _-s in the title, and the `pdf.pdf` is also strange and unnecessary and it is not so professional 😉  🙏